src/content/docs/fr/guides/deploy/zerops.mdx

```diff
@@ -0,0 +1,235 @@
+
+Pour indiquer à Zerops comment construire et exécuter votre site en utilisant l'adaptateur officiel [Astro Node.js](/fr/guides/integrations-guide/node/) en mode `standalone`, ajoutez un fichier `zerops.yml` à votre dépôt :
+
+<PackageManagerTabs>
+  <Fragment slot="npm">
+    ```yaml title="zerops.yml"
+    # voir https://docs.zerops.io/references/zeropsyml pour la référence complète
+    zerops:
+    - setup: app
+    build:
+    base: nodejs@20
+    buildCommands:
+    - npm i
+    - npm run build
+    deployFiles:
+    - dist
+    - package.json
+    - node_modules
+    run:
+    base: nodejs@20
+    ports:
+    - port: 3000
+    httpSupport: true
+    envVariables:
+    PORT: 3000
+    HOST: 0.0.0.0
+    start: npm start
+    ```
+  </Fragment>
+  <Fragment slot="pnpm">
+    ```yaml title="zerops.yml"
+    # voir https://docs.zerops.io/references/zeropsyml pour la référence complète
+    zerops:
+    - setup: app
+    build:
+    base: nodejs@20
+    buildCommands:
+    - pnpm i
+    - pnpm run build
+    deployFiles:
+    - dist
+    - package.json
+    - node_modules
+    run:
+    base: nodejs@20
+    ports:
+    - port: 3000
+    httpSupport: true
+    envVariables:
+    PORT: 3000
+    HOST: 0.0.0.0
+    start: pnpm start
+    ```
+  </Fragment>
+  <Fragment slot="yarn">
+    ```yaml title="zerops.yml"
+    # voir https://docs.zerops.io/references/zeropsyml pour la référence complète
+    zerops:
+    - setup: app
+    build:
+    base: nodejs@20
+    buildCommands:
+    - yarn
+    - yarn build
+    deployFiles:
+    - dist
+    - package.json
+    - node_modules
+    run:
+    base: nodejs@20
+    ports:
+    - port: 3000
+    httpSupport: true
+    envVariables:
+    PORT: 3000
+    HOST: 0.0.0.0
+    start: yarn start
+    ```
+  </Fragment>
+</PackageManagerTabs>
+
+Maintenant, vous pouvez [déclencher le pipeline de construction et de déploiement en utilisant le CLI Zerops](#déclencher-le-pipeline-en-utilisant-zerops-cli-zcli) ou en connectant le service `app` à votre dépôt [GitHub](https://docs.zerops.io/references/github-integration/) / [GitLab](https://docs.zerops.io/references/gitlab-integration) depuis le détail du service.
+
+## Déclencher le pipeline en utilisant Zerops CLI (zcli)
+
+<Steps>
+  1. Installez le CLI Zerops.
+      ```shell
+      # Pour télécharger le binaire zcli directement,
+      # utilisez https://github.com/zeropsio/zcli/releases
+      npm i -g @zerops/zcli
+      ```
+
+  2. Ouvrez [̀Settings > Access Token Management](https://app.zerops.io/settings/token-management) dans l'application Zerops et générez un nouveau jeton d'accès.
+
+  3. Connectez-vous en utilisant votre jeton d'accès avec la commande suivante :
+      ```shell
+      zcli login <token>
+      ```
+
+  4. Naviguez jusqu'à la racine de votre application (où se trouve `zerops.yml`) et exécutez la commande suivante pour déclencher le déploiement :
+      ```shell
+      zcli push
+      ```
+</Steps>
+
+## Ressources
+### Officiel
```

Comment:
Since this title refers to the previous title ("Ressources"), I think it is better to conjugate it.
```suggestion
### Officielles
```